### DISCUSSION TITLE
env variables for require_plex_login

### DISCUSSION THREAD

**Discussion by HMpugh** [LukeChannings/moviematch#104] [Q&A]
Attempting to setup requiring a plex login. What is the correct way to set up the docker-compose for that variable. I assumed it would be a boolean like I have below.

  moviematch:
   image: lukechannings/moviematch:latest
   container_name: moviematch
   environment:
    PLEX_URL: "<Plex URL>"
    PLEX_TOKEN: "<Plex Token>"
    LIBRARY_TITLE_FILTER: Movies
    REQUIRE_PLEX_LOGIN: true
   ports:
      - 8000:8000

**Comment by Manfred73** [comment]
I was looking to set this up as well, but the documentation does not clearly state the values of all the possible environment variables. I tried the value true as well, but it does not show me the Plex login option.
When accessing the url on port 8000 I get a login screen where I can only fill out my name and generate a room number.
I think it is connected to my Plex server, because I see movies that are in my Plex. But how can I verify it is actually connected to my Plex server? The log only shows listening on port 8000 and that I logged in.
And where is the configuration stored of my choices? Can this also be externally saved by configuring a volume?

**Reply by ninthwalker** [reply]
You have to use the beta versions for plex sso signin, but it was never finished and has bugs/security issues still. So not advised to use externally.
